### DISCUSSION TITLE
Zero to Nix, a new learning resource for Nix built using îles

### DISCUSSION THREAD

**Discussion by lucperkins** [ElMassimo/iles#229] [Show and tell]
Hello everyone! Just wanted to share something built with the lovely îles.

[Zero to Nix](https://zero-to-nix.com) is a learning resource for [Nix](https://nixos.org) from [Determinate Systems](https://determinate.systems). We built the site using îles, with all of our [MDX](https://mdxjs.com/) components in [Vue](https://vuejs.org) and [Tailwind](https://tailwindcss.com) for CSS.

Why did we choose îles? Pretty simple:

* We wanted MDX support for arbitrarily complex interactive components inside our Markdown
* We wanted to have our choice of platform for those interactive components
* We wanted to be able to easily integrate things like Tailwind (which îles accomplishes using Vite)
* îles has built-in content management via `useDocuments` and other functions, which meant that we didn't have to use something like [Contentlayer](https://contentlayer.dev), which is quite nice but we preferred using framework primitives instead.

Given the current feature velocity of [Astro](https://astro.build), we may end up switching to that at some point if the site continues to grow, but our experience with îles for going to market has been fantastic overall, and @ElMassimo has been extremely responsive. Our sincere thanks to Máximo and to the îles community 💯

**Comment by ElMassimo** [comment]
Great to hear, thanks for sharing Luc! 😃
